motor/inventory/v1/inventory.proto

```diff
@@ -136,6 +137,8 @@ message InventorySpec {
   map<string, cnquery.motor.vault.v1.Credential> credentials = 2;
   VaultConfiguration vault = 3;
   string credential_query = 4;
+  // optional: the upstream credentials to use for the inventory
+  mondoo.upstream.v1.ServiceAccountCredentials upstream_credentals = 5;
```

Comment:
I think that should be number greater than 15 https://developers.google.com/protocol-buffers/docs/proto3#assigning_field_numbers